public/css/icinga/menu.less

```diff
@@ -514,6 +518,28 @@ html.no-js  #toggle-sidebar {
   margin-right: 0;
 }
 
+#layout.sidebar-collapsed #menu .nav-level-1 > .nav-item.hover .nav-level-2 > .nav-item-header {
+  background-color: @menu-bg-color;
+  border-bottom: 1px solid @gray-light;
+  border-top-left-radius: .25em;
+  border-top-right-radius: .25em;
+
+  span {
+    font-weight: @font-weight-bold;
+    line-height: 2.167em;
+    padding: .5em .5em .5em 1.375em;
```

Comment:
Don't use line height and padding to achieve a desired height. Use a plain height instead. With #5363 there's a variable for this. (`@nav-item-height`)

To align the content, use flex box.